lua/telescope/builtin/__lsp.lua

```diff
@@ -359,7 +359,14 @@ lsp.document_symbols = function(opts)
       return
     end
 
-    local locations = vim.lsp.util.symbols_to_items(result or {}, opts.bufnr) or {}
+    local locations = {}
```

Comment:
```suggestion
    local locations
```

There's some linting errors related to this and similarly down below.